language/move-lang/src/parser/syntax.rs

```diff
@@ -435,6 +435,21 @@ fn parse_byte_string<'input>(tokens: &mut Lexer<'input>) -> Result<Value_, Error
     Ok(value_)
 }
 
+// Create an error for an integer literal that is too big to fit in its type.
+// This assumes that the literal is the current token.
+fn num_too_big_error<'input>(tokens: &mut Lexer<'input>, type_description: &str) -> Error {
```

Comment:
Heads up you'll need to switch this to `&mut Lexer<'_>` after #7654

Reply:
Oh nice! I like it much better without all those named lifetimes....